tfe/resource_tfe_team_token.go

```diff
@@ -0,0 +1,84 @@
+package tfe
+
+import (
+	"fmt"
+	"log"
+
+	tfe "github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceTFETeamToken() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceTFETeamTokenCreate,
+		Read:   resourceTFETeamTokenRead,
+		Delete: resourceTFETeamTokenDelete,
+
+		Schema: map[string]*schema.Schema{
+			"team_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"token": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
```

Comment:
I believe this is a sensitive token? if so should this have `Sensitive: true`

Reply:
Fixed in next commit.